indico/MaKaC/services/implementation/contribution.py

```diff
@@ -314,11 +307,11 @@ def _getAnswer(self):
             if user["_type"] == "Avatar":  # new speaker
                 part = self._newParticipant(principal_from_fossil(user, allow_pending=True))
             elif user["_type"] == "ContributionParticipation":  # adding existing author to speaker
-                part = self._contribution.getAuthorById(user["id"])
-                self._contribution.addSpeaker(part)
+                author_index_author_id = "{} {} {}".format(user['familyName'], user['firstName'], user['email']).lower()
```

Comment:
That's useful thanks.
`author_index_id` might refer to the ID of the index itself, no? This is supposed to be the ID of the author inside the author_index.